addons/mail/static/tests/owl/hooks/use_store_compare_keys_tests.js

```diff
@@ -0,0 +1,218 @@
+odoo.define('mail.hooks.useStoreCompareKeysTests', function (require) {
+'use strict';
+
+const { proxyComparator, proxyComparatorDeep, useStoreCompareKeys } = require('mail.hooks.useStoreCompareKeys');
+
+const { Component, QWeb, Store } = owl;
+const { xml } = owl.tags;
+
+const {
+    beforeEach: utilsBeforeEach,
+    afterEach: utilsAfterEach,
+    afterNextRender,
+    nextAnimationFrame,
+} = require('mail.messagingTestUtils');
+
+QUnit.module('mail.messaging', {}, function () {
+QUnit.module('hooks', {}, function () {
+QUnit.module('useStoreCompareKeysTests', {
+    beforeEach() {
+        utilsBeforeEach(this);
```

Comment:
don't need, I think

Reply:
I copied from another test, probably not needed indeed.